src/ensembl/src/content/app/entity-viewer/shared/rest/rest-data-fetchers/proteinData.ts

```diff
@@ -0,0 +1,48 @@
+import { restProteinStatsAdaptor } from '../rest-adaptors/rest-protein-adaptor';
+import { TranscriptInResponse } from './transcriptData';
+
+export type XrefsInResponse = {
+  display_id: string;
+}[];
+
+export type ProteinStatsInResponse = {
+  pdbs: number;
+  ligands: number;
+  interaction_partners: number;
+  annotations: number;
+};
+
+export type UniProtSummaryStats = {
+  [key: string]: ProteinStatsInResponse;
+};
+
+export type ProteinStats = Pick<
+  ProteinStatsInResponse,
+  'ligands' | 'annotations'
+> & {
+  structures: number;
+  interactions: number;
+};
+
+export const fetchProteinSummaryStats = async (
+  transcriptId: string,
+  signal?: AbortSignal
+): Promise<ProteinStats> => {
+  const transcriptUrl = `http://rest.ensembl.org/lookup/id/${transcriptId}?expand=1;content-type=application/json`;
+  const transcript: TranscriptInResponse = (await fetch(transcriptUrl, {
+    signal
+  }).then((response) => response.json())) as TranscriptInResponse;
```

Comment:
Here and below — no need to take the await fetch expression in parentheses and annotate it with `as TranscriptInResponse`. This should work just fine:

```
const transcript: TranscriptInResponse = await fetch(transcriptUrl, {
    signal
  }).then((response) => response.json());
```

Reply:
✔️